packages/@okta/vuepress-theme-prose/components/Sidebar.vue

```diff
@@ -1,43 +1,54 @@
 <template>
   <aside class="landing-navigation">
     <ul class="landing">
-      <SidebarItem v-for="link in navigation" :key="link.title" :link="link" />
+      <ul class="sections">
+        <SidebarItem
+          v-for="link in navBarItems"
```

Comment:
this might be breaking active sidebar item highlight on URL change with stack selector (e.g. https://developer.okta.com/docs/guides/sign-into-spa/react/handle-callback/)

Reply:
it looks like this is happening b/c `navBarItems` is not getting updated on Sidebar update